.gitignore

```diff
@@ -0,0 +1,2 @@
+# .gitignore
+__pycache__

```

Comment:
Also add pipfile.lock to gitignore

Reply:
Also add .pyc. I still see pyc files committed.

Reply:
Use this standard python gitignore https://github.com/github/gitignore/blob/master/Python.gitignore